src/options.js

```diff
@@ -72,6 +72,16 @@ function normalize(options) {
     );
   }
 
+  /* istanbul ignore if */
+  if (typeof normalized.proseWrap === "boolean") {
+    normalized.proseWrap = normalized.proseWrap ? "always" : "never";
+
+    console.warn(
```

Comment:
Do we still use `deprecated.js`? It seems to be outdated.

Reply:
It seems it's only used for `useFlowParser` warning.

I think we can refactor those option warnings/redirections using #3352.